src/client.rs

```diff
@@ -138,14 +166,14 @@ impl InstrumentedClientRequest {
         R: Future<Output = AwcResult>,
     {
         let tracer = global::tracer("actix-client");
-        let mut attributes = vec![
+        self.attrs.extend(vec![
```

Comment:
we may be able to use an iterator rather than a new vec here to avoid the additional static size allocation, but doesn't have to be in this PR